ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -616,6 +617,20 @@ errdisable:
 
 ### Filters
 
+#### Dynamic Prefix Lists
+
+```yaml
+dynamic_prefix_lists:
+  - name: < dynamic_prefix_list_name_1 >
+    match_map: < route_map >
+    prefix_list:
+      ipv4: < prefix_list >
```

Comment:
We can have both ipv4 and ipv6 defined for the same prefix-list. I just tested it now: 
<img width="178" alt="image" src="https://user-images.githubusercontent.com/88734745/161983743-9b3b728a-1f44-4033-9423-728ecf2ea02a.png">

Reply:
I just tested on EOS this is allowed to define a prefix-list for both address types in the same dynamic prefix-list.  I will update the test and the templates to allow/document it.